src/css/header.scss

```diff
@@ -81,3 +81,50 @@
     margin-top: -18px;
   }
 }
+
+#new-desktop-header {
+  height: 88px;
+  max-width: 1440px;
```

Comment:
Can we use `$max-content-width - 7` for this value?

Reply:
happy to make the change, but why `- 7`?

Reply:
Just because it looks like the variable `$max-content-width` is `1447`

Reply:
ah I think I meant to use `$max-content-width` and just forgot to change it